clients/gen/common.sh

```diff
@@ -17,7 +17,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-OPENAPI_GENERATOR_CLI_VER=5.1.1
+OPENAPI_GENERATOR_CLI_VER=5.2.1
```

Comment:
let's use the latest version i.e. `5.3.0`

Reply:
Done